hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestContainerBalancerOperations.java

```diff
@@ -103,7 +100,7 @@ public void testContainerBalancerCLIOperations() throws Exception {
     // TODO: this is a temporary implementation for now
     // modify this after balancer is fully completed
     try {
-      Thread.sleep(100);
+      Thread.sleep(200000);
```

Comment:
We can reduce this sleep time by reducing some config values for the test. In containerBalancerTask you can see this:
```
if (config.getTriggerDuEnable()) {
...
         long nodeReportInterval =
              ozoneConfiguration.getTimeDuration(HDDS_NODE_REPORT_INTERVAL,
                  HDDS_NODE_REPORT_INTERVAL_DEFAULT, TimeUnit.MILLISECONDS);
          ...
          long sleepTime = 3 * nodeReportInterval;
          ...
}
```
We can set HDDS_NODE_REPORT_INTERVAL to 5s or so (by default it is 60s) in the setup. This case we would have to sleep only for 5*3=15secs instead of 3mins